index.js

```diff
@@ -1,3 +1,165 @@
 'use strict';
 
-console.log('HELLO!');
+const docker = require('./lib/docker');
+const fs = require('fs');
+const yaml = require('js-yaml');
+const path = require('path');
+
+function getOptions() {
+    // check process arguments
+    const args = require('yargs')
+    .usage('Usage: $0 [command] [options]')
+    .options({
+        n: {
+            alias: 'num-workers',
+            default: -1,
+            describe: 'number of workers to start',
+            nargs: 1,
+            global: true
+        },
+        c: {
+            alias: 'config',
+            default: './config.yaml',
+            describe: 'YAML-formatted configuration file',
+            type: 'string',
+            nargs: 1,
+            global: true
+        },
+        verbose: {
+            default: false,
+            describe: 'be verbose',
+            type: 'boolean',
+            global: true
+        },
+        v: {
+            alias: 'version',
+            default: false,
+            describe: 'print the service\'s version and exit',
+            type: 'boolean',
+            global: true
+        }
+    })
+    .command('docker-start', 'starts the service in a Docker container')
```

Comment:
Perhaps change it into `run` or something similar? The `docker-` prefix seems a bit superfluous now.